niworkflows/anat/skullstrip.py

```diff
@@ -23,7 +23,8 @@ def afni_wf(name='AFNISkullStripWorkflow'):
     outputnode = pe.Node(niu.IdentityInterface(
         fields=['bias_corrected', 'out_file', 'out_mask', 'bias_image']), name='outputnode')
 
-    inu_n4 = pe.Node(ants.N4BiasFieldCorrection(dimension=3, save_bias=True),
+    inu_n4 = pe.Node(ants.N4BiasFieldCorrection(
+        dimension=3, save_bias=True, num_threads=n4_nthreads), num_threads=n4_nthreads,
                      name='CorrectINU')
```

Comment:
`Node` takes an `n_procs` parameter, not `num_threads`. Also PEP8 fix for white space:

```Python
inu_n4 = pe.Node(
    ants.N4BiasFieldCorrection(dimension=3, save_bias=True, num_threads=n4_nthreads),
    n_procs=n4_threads,
    name='inu_n4')
```

And should we be copying the header, as in [fmriprep](https://github.com/poldracklab/fmriprep/blob/master/fmriprep/workflows/util.py#L64-L79)? Maybe it would make sense to move that here, and thread in the `n4_threads`?